homeassistant/components/cloud/client.py

```diff
@@ -85,28 +87,44 @@ def remote_autostart(self) -> bool:
     async def get_alexa_config(self) -> alexa_config.CloudAlexaConfig:
```

Comment:
we have a singleton decorator for this that stores this in `hass.data`. Would be nice if we could have a singleton decorator for classes too.

Reply:
https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/singleton.py